stable/gce-ingress/templates/deployment-backend.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: apps/v1beta2
```

Comment:
use v1 api

Reply:
ahhh good catch!  the `helm create` templates must be wayyyy behind.